tests/data_platform/utils/test_storage.py

```
@@ -0,0 +1,157 @@
+        dedupe_session=dedupe_session,
+    )
+
+    assert result.kept == 1
+    assert result.skipped == 1
+    assert bluesky_storage.load_seen_uris(run_dir) == {
+        "at://did:plc:ex/app.bsky.feed.post/a1",
+        "at://did:plc:ex/app.bsky.feed.post/a2",
+    }
+
+
+def test_append_deduped_records_skips_prior_run_duplicates(data_root) -> None:
```

Ruff (0.16.1)

[warning] Unused function argument: `data_root`

(ARG001)